getting-started/login-for-multi-account-users.md

```diff
@@ -0,0 +1,29 @@
+# Login for multi-account users
```

Comment:
Heading of the section should be about the feature to add same user in multiple accounts. Then we can explain how it works in short and then we can have the login flow explanation section.

Reply:
done